src/atomic/pag.login/login.component.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { useTheme } from 'styled-components';
+
+import { AtomSeparator } from '../atm.separator/separator.component';
+import { AtomTitle } from '../atm.title/title.component';
+import { MoleculeCallToActionButton } from '../mol.button/button.component';
+import { MoleculeNamedInput } from '../mol.named-input/named-input.component';
+
+import { StylePageLoginContainer } from './login.component.style';
+
+export const PageLogin: React.FC = () => {
```

Comment:
Vc está usando essa `PageLogin` somente como componente visual? Se for eu diria pra alterar o nome, `LoginForm` talvez, pq vc ta utilizando no `ScreenLogin` e pelo nome os 2 me parecem a mesma coisa..

 Se não for somente visual eu diria pra remover o `ScreenLogin` e usar somente o `PageLogin`, ou o contrario

Reply:
A minha ideia seria o `ScreenLogin` receber a lógica de enviar login. Já que no Atomic o template(page no projeto) não tem informação dinâmica só o estilo. O que acha?